.circleci/config.yml

```diff
@@ -357,13 +373,10 @@ jobs:
   #         command: |
   #           set -x
   #           export PATH=~/.local/bin:$PATH
-
```

Comment:
super nitpick but maybe add these whitespaces back in so this comment remains easier to read? with `#` in front if a linter complains